kudu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM oryxprod/build:20190521.9 as main
+FROM mcr.microsoft.com/oryx/build:20190617.2 as main
```

Comment:
Tested oryx build on this image, although in the oryx help said `--platform` is supported, but still running into an issue where `Unrecognized option '--platform' /opt/Kudu/Scripts/starter.sh oryx build /home/site/wwwroot -o /home/site/wwwroot --platform python --platform-version 3.6 -i /tmp/8d7019a264a79e8 -p packagedir=.python_packages/lib/python3.6/site-packages`
when running `oryx build /home/site/wwwroot -o /home/site/wwwroot --platform python --platform-version 3.6 -i /tmp/8d7019a264a79e8 -p packagedir=.python_packages/lib/python3.6/site-packages`

Reply:
The latest image 20190704.2 fix this issue, please check.

Reply:
what is the output of oryx --version?
Either way I'll bump the version